packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -64,10 +64,11 @@ export const GraphNG: React.FC<GraphNGProps> = ({
   const legendItemsRef = useRef<LegendItem[]>([]);
   const hasLegend = useRef(legend && legend.displayMode !== LegendDisplayMode.Hidden);
   const alignedFrame = alignedFrameWithGapTest.frame;
-  const compareFrames = useCallback(
-    (a: DataFrame, b: DataFrame) => compareDataFrameStructures(a, b, ['min', 'max']),
-    []
-  );
+
+  const compareFrames = useCallback((a: DataFrame, b: DataFrame) => {
+    return compareDataFrameStructures(a, b, ['min', 'max']);
```

Comment:
I do not think you need to ignore min max anymore as they set dynamically by applyFieldOverrides anymore